docs/workflows/transitions/preprocess/preprocess/make_predictions.py

```diff
@@ -15,31 +15,60 @@ def make_predictions(las_client, event):
     document_id = event['documentId']
     model_id = os.environ['MODEL_ID']
     form_config_id = os.environ['FORM_CONFIG_ASSET_ID']
-    
+
     logging.info(f'Processing event:')
     logging.info(json.dumps(event, indent=2))
 
     form_config_asset = las_client.get_asset(form_config_id)
     form_config = json.loads(base64.b64decode(form_config_asset['content']))
-    
+
+    model_metadata = las_client.get_model(model_id).get('metadata', {})
+    logging.info(f'model metadata: {model_metadata}')
+
     output = {}
     needs_validation = True
 
+    labels = get_labels(form_config['config']['fields'])
+    no_empty_prediction_fields = set()
+
     if not (predictions := event.get('predictions')):
-        try:
-            predictions = las_client.create_prediction(document_id, model_id).get('predictions')
-            logging.info(f'Created predictions {predictions}')
-        except Exception as e:
-            logging.exception(e)
-            predictions = []
-            output = {'predictions': predictions}
+        start_page = 0
+        predictions = []
+        while start_page is not None and start_page < model_metadata.get('maxPredictionPages', 100):
+            try:
+                current_prediction = las_client.create_prediction(
+                    document_id=document_id,
+                    model_id=model_id,
+                    preprocess_config={'startPage': start_page, 'maxPages': 3, 'imageQuality': 'HIGH'},
```

Comment:
I suggest we get the maxPages here from the model's preprocess config.

Reply:
Couldn't we get the entire preprocess config from the model and then just add startPage?